paasta_tools/kubernetes_tools.py

```diff
@@ -610,9 +608,9 @@ def __init__(
         # We need to support apiextensions /v1 and /v1beta1 in order
         # to make our upgrade to k8s 1.22 smooth, otherwise
         # updating the CRDs make this script fail
-        self.apiextensions_v1_beta1 = kube_client.ApiextensionsV1beta1Api(
-            self.api_client
-        )
+        # self.apiextensions_v1_beta1 = kube_client.ApiextensionsV1beta1Api(
```

Comment:
was it dead code?

Reply:
I removed it now

Reply:
if we need this code again we can grab it from git history :)